CRM/Contribute/BAO/Contribution.php

```diff
@@ -4145,6 +4146,27 @@ public static function getPaymentInfo($id, $component, $getTrxnInfo = FALSE, $us
           }
           $paidByLabel .= " ({$creditCardType}{$pantruncation})";
         }
+
+        // show payment edit link only for payments done via backoffice form
```

Comment:
Question: do any online payment processor plugin currently support changes to the value of a transaction, @eileenmcnaughton ? If so, we should allow those payments to be editted. JMA is aiming to support this for Authorize.net this summer assuming client funding comes through.